src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13965,10 +13965,18 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                     ShowContinueErrorTimeStamp(state, "");
                     ShowContinueError(state, format("  Iteration limit [{}] exceeded in calculating OU evaporating temperature", MaxIter));
                 } else if (SolFla == -2) {
-                    // demand < capacity at both endpoints of the Te range, assuming f(x) is roughly monotonic than this is the low load case
-                    assert(f(T_suction) < 0);
-                    // TeTol is added to prevent the final updated Te to go out of bounds
-                    SmallLoadTe = 6 + TeTol; // MinOutdoorUnitTe; //SmallLoadTe( Te'_new ) is constant during iterations
+                    if (f(T_suction) < 0) {
+                        // demand < capacity at both endpoints of the Te range, assuming f(x) is roughly monotonic than this is the low load case
+                        // TeTol is added to prevent the final updated Te to go out of bounds
+                        SmallLoadTe = 6 + TeTol; // MinOutdoorUnitTe; //SmallLoadTe( Te'_new ) is constant during iterations
+                    } else {
+                        // demand > capacity at both endpoints of the Te range, take the end point x where f(x) is closer to zero
+                        if (f(MinOutdoorUnitTe) > f(T_suction)) { // f(T_suction > 0, not equal as SolFla will not be -2
+                            SmallLoadTe = T_suction;
+                        } else {
+                            SmallLoadTe = MinOutdoorUnitTe;
+                        }
+                    }
```

Comment:
The only thing missing here is the ShowWarningMessage to identify how often these `-2` cases occur. Also, when `SolFla == -1`, the bounds (i.e., MinOutdoorUnitTe and T_suction) "could" have been tightened to allow a solution to be found because a solution was possible but the max iteration limit was hit. However, the `-1` type of failure is rare and should be handled individually as needed.

Reply:
Questioning whether this should be `SmallLoadTe = 6 + TeTol; // MinOutdoorUnitTe` or SmallLoadTe = MinOutdoorUnitTe since:

                MinOutdoorUnitTe = this->refrig->getSatTemperature(state, max(min(MinOutdoorUnitPe, RefPHigh), RefPLow), RoutineName);
                // Te can't be smaller than user input lower bound
                MinOutdoorUnitTe = max(this->IUEvapTempLow, MinOutdoorUnitTe);